data_steward/test/unit_test/validation_test.py

```diff
@@ -9,13 +9,15 @@
 from validation import main
 import resources
 import common
+import copy
```

Comment:
Is this used?

Reply:
No. I had that for a test I wrote that became redundant. Will remove it.